build-scripts/import_ocw.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [ -f .env ]
+then
+  export $(cat .env | sed 's/#.*//g' | xargs)
```

Comment:
could this just be `source .env` ?

Reply:
I tried changing this line to that and it worked correctly, but not sure if there's a bash thing to be concerned about there (bash always concerns me haha)

Reply:
so after looking up the difference between `source` and `export` (which I must admit I was tragically ignorant of) it seems we probably want to export these variables so that they can be read from within `ocw-to-hugo`. seems like there should be a nicer way to export a bunch of variables, but I think this is good